vyos/vyos-build
# T5468: Remove unused manpages to free up space

## Change Summary

We dont need any manpages on the system since man-binary is missing.
This will also free up some space.

## Types of changes

- [x] Other (please describe): Removes functionality who cannot be used anyway since man-binary is missing.

## Related Task(s)

* https://vyos.dev/T5468

## Component(s) name

manpages

## Proposed changes

Delete directory /usr/share/man to free up some space since the content cannot be used anyway since man-binary is missing. Approx 13 megabyte (compressed) can be freed which also shrinks iso-file.

Also a re-numbering of the scripts in the build-directory (`/data/live-build-config/hooks/live`) have been issued (refactoring).

New feature (deleting manpages) have been created as file `81-delete-manpages.chroot`.

## How to test

Attempt to run `man` shows that the binary is missing.

Verify that the manpages directory over at /usr/share/man is removed by issuing:

```
ls -la /usr/share/man
```

`du -sm /usr/share/man` can also be used to verify amount of space saved (runned on iso prior to this merge).

## Checklist:
